tests/utils/stl/test/tests.py

```diff
@@ -135,6 +141,8 @@ def _configure_cxx(self, lit_config, envlst_entry, default_cxx):
         link_flags.extend(envlst_entry.getEnvVal('PM_LINK', '').split())
 
         if ('clang'.casefold() in os.path.basename(cxx).casefold()):
+            flags.extend(map(lambda x : '/clang:' + x, envlst_entry.getEnvVal('PM_CLANG', '').split()))

```

Comment:
I'm not sure how I feel about this new option. I think it's a little bit too much magic. Why not just add the `/clang:*` options to `PM_CL` in the test configurations you need them in?

I don't feel super strongly about this so convincing me should be easy.